students/mia/users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+# Create your models here.
+
+class User(models.Model): 
+    name          = models.CharField(max_length=50)
+    email         = models.EmailField(max_length =100, unique=True)
```

Comment:
email 또한 `EmailField`가 아닌 `CharField`로 작성해주세요!
```suggestion
    email         = models.CharField(max_length =100, unique=True)
```